merchants/models.py

```diff
@@ -47,3 +51,242 @@ class PaymentProduct(models.Model):
 
     def __str__(self) -> str:
         return str(self.id)
+
+
+class PaymentType(models.TextChoices):
+    ON_SITE = "ON_SITE", _("On site")
+    ONLINE = "ONLINE", _("Online")
+    INVOICE = "INVOICE", _("Invoice")
+
+
+class PaymentStatus(models.TextChoices):
+    DRAFT = "DRAFT", _("Draft")
+    EXPIRED = "EXPIRED", _("Expired")
+    CANCELLED = "CANCELLED", _("Cancelled")
+    PAID = "PAID", _("Paid")
+    PAID_MANUALLY = "PAID_MANUALLY", _("Paid manually")
+    REFUNDED = "REFUNDED", _("Refunded")
+
+
+class Language(models.TextChoices):
+    FI = "fi", _("Finnish")
+    SV = "sv", _("Swedish")
+    EN = "en", _("English")
+
+
+class PaymentOrder(models.Model):
+    order_id = models.UUIDField(
+        verbose_name=_("Order ID"),
+        help_text=_("Talpa eCommerce order ID"),
```

Comment:
This could be more common model for other eCommerce platforms so I would get rid off "Talpa"

Reply:
Removed 👍